jsx/src/components/ServerDashboard/ServerDashboard.jsx

```diff
@@ -176,6 +175,14 @@ const ServerDashboard = (props) => {
     );
   };
 
+  let servers = user_data.flatMap((user) => {
+    let userServers = Object.values({
+      "": user.server,
```

Comment:
I think this could be the way to avoid issues below where it is assumed that you have a default server.

```suggestion
      "": user.server || {},
```

Reply:
This code suggestion makes the tests pass, see https://github.com/consideRatio/jupyterhub/runs/5511339074?check_suite_focus=true.